### ISSUE TITLE
Continuity

### ISSUE THREAD

**Issue by ddeboer** [basho/sidejob#16]
For your information: with the demise of Basho, we decided to fork this repo to be able to keep maintaining it. Head to https://github.com/erlangpack/sidejob for an updated version. Also available on Hex.

**Comment by russelldb** [comment]
A post on the riak ML might be helpful too

**Comment by ddeboer** [comment]
Done. 

Any of the original maintainers still reading this? Please let know: I’d like to _move_ this repository in order to prevent confusion on where to head for issues and updates.

**Comment by russelldb** [comment]
I worked at Basho, I don't know how you'd move it. I imagine it "belongs" to Basho the org. Not sure who is left to ask to move it. AFAIK the company is a zombie. Maybe Adam Wray or Dave McCrory could be persuaded to move the repo (as they did for lager and webmachine in the past.) Maybe email them directly?

**Comment by ddeboer** [comment]
You need to be a repository admin for that, so one of [these people](https://github.com/orgs/basho/people) (there may be more).

**Comment by russelldb** [comment]
Well, I can see a lot of people in that list (in fact I'm in it privately) but only one of them still works for Basho afaik. Not going to out him on this thread, but I will ping him in a backchannel and see if he can help you.

**Comment by ddeboer** [comment]
Thanks @russelldb!

**Comment by russelldb** [comment]
I just looked at the list logged out of github, lol no, all those people work elsewhere and have for quite some time. How weird.

**Comment by ddeboer** [comment]
When logged out, you only see that members that made their membership public. You can choose to hide it (the default), but then you see that if you’re one of them and logged in. Anyway, we need an admin/owner in the basho GitHub group.

**Comment by russelldb** [comment]
Yeah, I just spoke to him. He can't help. I'm sorry but there are _no_ owners/engineers/devops/technical staff of any kind left at Basho as far as I can tell. You can reach out to Dave McCrory or Adam Wray (CEO and CTO respectively) directly. That would be my first call. mccrory@basho.com was the CTO's email. If it is deactivated maybe try twitter https://twitter.com/mccrory?

**Comment by cmeiklejohn** [comment]
For what it's worth, @licenser, @tsloughter and myself (who, depend heavily on Riak libraries) have created forks that are Erlang 19+ compatible here: https://github.com/Kyorai.  We are also maintaining CRDT implementations and a fork of Riak Core under the Lasp organization.

**Comment by ddeboer** [comment]
@russelldb Thanks for checking! I’ll try to contact them.

@cmeiklejohn I don’t see any sidejob fork at the organization that you linked to.

**Comment by cmeiklejohn** [comment]
@ddeboer We only forked the libraries that we took a hard dependency on and needed to modify for Erlang 19+ support, that is why we didn't touch sidejob.
